code/src/styling/range-slider.css

```diff
@@ -0,0 +1,130 @@
+    background-color: transparent;
+  }
+  input[type=range]::-webkit-slider-runnable-track {
+    margin-top: 1rem;
+    width: 100%;
+    height: 8px;
+    cursor: pointer;
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+    background: radial-gradient(circle, #db1616, #8b0000, rgb(97, 2, 2));
+    border-radius: 25px;
+    border: 0px solid #000101;
+  }
+  input[type=range]::-webkit-slider-thumb {
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+    border-radius: 7px;
+    border: 0px solid #000000;
+    height: 80px;
+    width: 80px;
+    margin-top: -2rem;
+    background-image: url("https://www.svgrepo.com/show/403170/dagger.svg"), radial-gradient(ellipse, white, #ffffff91, #ff898961, #8b000000, rgba(97, 2, 2, 0));
+    transform: rotate(45deg);
+    background-size: cover;
+    cursor: pointer;
+    -webkit-appearance: none;
+  }
+  
+  input[type=range]:focus::-webkit-slider-runnable-track {
+    background: radial-gradient(circle, #db1616, #8b0000, rgb(97, 2, 2));
+  }
+  input[type=range]::-moz-range-track {
+    width: 100%;
+    height: 12.8px;
+    cursor: pointer;
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+    background: radial-gradient(circle, #db1616, #8b0000, rgb(97, 2, 2));
+    border-radius: 25px;
+    border: 0px solid #000101;
+  }
+
+  input[type=range]::-moz-range-thumb {
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+    border: 0px solid #000000;
+    height: 20px;
+    width: 39px;
+    border-radius: 7px;
+    background: transparent;
+    cursor: pointer;
+    height: 80px;
+    width: 80px;
+    margin-top: -2rem;
+    background-image: url("https://www.svgrepo.com/show/403170/dagger.svg");
+    transform: rotate(45deg);
+    background-size: cover;
+}
+
+
+
+  input[type=range]::-ms-track {
+    width: 100%;
+    height: 12.8px;
+    cursor: pointer;
+    background: transparent;
+    border-color: transparent;
+    border-width: 39px 0;
+    color: transparent;
+  }
+  input[type=range]::-ms-fill-lower {
+    background: radial-gradient(circle, #db1616, #8b0000, rgb(97, 2, 2));
+    border: 0px solid #000101;
+    border-radius: 50px;
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+  }
+  input[type=range]::-ms-fill-upper {
+    background: radial-gradient(circle, #db1616, #8b0000, rgb(97, 2, 2));
+    border: 0px solid #000101;
+    border-radius: 50px;
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+  }
+  input[type=range]::-ms-thumb {
+    box-shadow: 0px 0px 0px #000000, 0px 0px 0px #0d0d0d;
+    border: 0px solid #000000;
+    height: 20px;
+    width: 39px;
+    border-radius: 7px;
+    background: #65001c;
+    cursor: pointer;
+  }
+  input[type=range]:focus::-ms-fill-lower {
+    background: radial-gradient(circle, #db1616, #8b0000, rgb(97, 2, 2));
+  }
+  input[type=range]:focus::-ms-fill-upper {
+    background: radial-gradient(circle, #db1616, #8b0000, rgb(97, 2, 2));
+  }
+  
+  body {
+    padding: 30px;
+  }
+  
```

Comment:
wow you have put a lot of effort for styling the range slider, and it is very cool! good thing that you have divided the css to different files, so it is easier to follow.